src/ui/Settings/Settings.ts

```diff
@@ -25,7 +25,6 @@ export interface ISettingsOptions {
  * This component can reference several components to populate its popup menu:
  * - {@link AdvancedSearch}
  * - {@link ExportToExcel}
- * - {@link FollowItem}
```

Comment:
I removed it because I realized it rather seems like a result template component.